.github/workflows/update-renovate-pr.yml

```diff
@@ -0,0 +1,48 @@
+---
+# Renovate Pull Requests are not updating the license files in the licenses folder.
+# This workflow will checkout the renovate PR, update the license files, and push the changes back to the PR.
+name: update-renovate-pr
+
+on:
+  push:
+    branches:
+      - renovate/**
+    paths-ignore:
+      - licenses/**
+
+permissions:
+  contents: read
+
+jobs:
+  update-renovate-pr:
+    permissions:
+      contents: write
+    runs-on: ubuntu-latest
+    steps:
+      - uses: actions/checkout@v4
+
+      - uses: ./.github/workflows/gradle-goal
+        with:
+          command: "./gradlew updateLicensesAndNotice"
+
+      - name: Debug diff
+        run: git status --porcelain
+
+      - name: Check if NOTICE.txt has changed
```

Comment:
we need to check if there is any change in the whole `licenses` folder here, there is no `NOTICE.txt`, the `NOTICE` at the project root is maintained manually.

Reply:
C&P from https://github.com/elastic/apm-server/blob/main/.github/workflows/update-dependabot-pr.yml :)

Reply:
https://github.com/elastic/elastic-otel-java/pull/390/commits/faf9b581abe3bb6e6eae1f04dc936b6b7ad3847e